theta_dup/include/theta_sketch_dup.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package datasketches_pb;
+
+message Update_theta_sketch_dup {
+  uint32 preamble_longs = 1;
+  uint32 serial_version = 2;
+  uint32 sketch_type = 3;
+  uint32 rf = 4;
+  uint32 lg_nom_size = 5;
+  uint32 lg_cur_size = 6;
+  uint32 flags_byte = 7;
+  uint32 seed_hash = 8;
+  uint32 num_keys = 9;
+  double p = 10;
+  uint64 theta = 11;
+
+  message hash_map_count {
+    uint64 hash_val = 1; 
+    uint64 cnt = 2;
```

Comment:
Counts should generally by signed, as a negative number is easier to detect and tackle than an overflow.

Also, please use the full word `count`, instead of the abbreviation.